docs/runtime/transfer_learning.md

```diff
@@ -0,0 +1,199 @@
+```
+For more information about feature extraction and fine tuning, please refer to [transfer learning tutorial](https://www.tensorflow.org/tutorials/images/transfer_learning).
+
+## Convert tflite model to circle model
+In the moment of writing, the `circle+` format is developed so it will the other possibility to handle model training.
+Note that `one-import` can be found in the installaion directory (determined by `-DCMAKE_INSTALL_PREFIX` cmake flag which can be added to `./nnfw configure` command).
+```bash
+one-import tflite -i customized_mobilenetv2.tflite -o customized_mobilenetv2.circle
+```
+
+## Run training
+Let's train the model using `onert_train`. In this case we want to test only 2 last layers: `Mean` and `FullyConnected` (created from `GlobalAveragePooling2D` and `Dense` in TensorFlow) Note that the other possibility is using `circle+` capabilities.
+NOTE TO REMOVE: Due to problems with exporing, the training was launched on https://github.com/Samsung/ONE/pull/13226
+```bash
+onert_train \
+--modelfile customized_mobilenetv2.circle \
+--epoch 5 \
+--loss 1 \                # mean squared error
+--loss_reduction_type 1 \ # sum over batch size
+--optimizer 2 \           # adam
+--learning_rate 0.0001 \
+--batch_size 32 \
+--num_of_trainable_ops 5 \
+--load_expected:raw cats_and_dogs.output.bin \
+--load_input:raw cats_and_dogs.input.bin \
+--export_path customized_mobilenetv2_trained.circle
+```
+The result of training:
+```
+Model Expected Filename cats_and_dogs.output.bin
+Model Input Filename cats_and_dogs.input.bin
+Model Filename customized_mobilenetv2.circle
+== training parameter ==
+- learning_rate        = 0.0001
+- batch_size           = 32
+- loss_info            = {loss = mean squared error, reduction = sum over batch size}
+- optimizer            = adam
+- num_of_trainable_ops = 10
+========================
+Epoch 1/5 - time: 532.378ms/step - loss: [0] 0.1567
+Epoch 2/5 - time: 531.966ms/step - loss: [0] 0.0417
+Epoch 3/5 - time: 532.795ms/step - loss: [0] 0.0104
+Epoch 4/5 - time: 532.607ms/step - loss: [0] 0.0053
+Epoch 5/5 - time: 532.567ms/step - loss: [0] 0.0049
+===================================
+MODEL_LOAD   takes 4.3720 ms
+PREPARE      takes 192.9690 ms
+EXECUTE      takes 165537.7600 ms
+- Epoch 1      takes 33007.4380 ms
+- Epoch 2      takes 32981.8970 ms
+- Epoch 3      takes 33033.2690 ms
+- Epoch 4      takes 33021.6380 ms
+- Epoch 5      takes 33019.1300 ms
+===================================
+```
+## Test inference
+In this tutorial the inference will be launched via `Python API`.
+### Convert the trained model to nnpackage format
+The current version of `Python API` expect models in Python nnpackge format. The conversion can be done with `tools/nnpackage_tool/model2nnpkg/model2nnpkg.py` script.
+```bash
+python model2nnpkg.py -m customized_mobilenetv2_trained.circle
+```
+### Run inference via Python API
+PRE: Install nnfwapi package
+```
+pip install -i https://test.pypi.org/simple/ nnfwapi==0.1.1
```

Comment:
I have a problem to build Python API from scratch, so I've finally used version from pypi. The problems are described in https://github.com/Samsung/ONE/issues/13332.